airbyte-lib/examples/run_test_source.py

```diff
@@ -14,7 +14,7 @@
 os.environ["AIRBYTE_LOCAL_REGISTRY"] = "./tests/integration_tests/fixtures/registry.json"
 
 source = ab.get_connector("source-test", config={"apiKey": "test"})
-cache = ab.get_in_memory_cache()
+cache = ab.new_local_cache(source_catalog=source.configured_catalog)
```

Comment:
Caches need a source catalog now, FYI. This informs data type decisions.

Reply:
Update: this is now captured in the read() method so it is not needed in constructors and factory functions.